### PR TITLE
Enable bookinfo automation for k8s native rules.

### PR DESCRIPTION
**Pull request by craigyam**
Update the bookinfo script and yaml files to use the k8s
native rules. Skip over Gremlin and traffic tests as they
are not supported with the k8s native rules.

### CODE REVIEW

File: testing/kubernetes/bookinfo.yaml [hunk 1]
```diff
@@ -53,18 +53,16 @@ spec:
...
+        - name: A8_RULES_ADAPTER
+          value: kubernetes
         - name: A8_KUBERNETES_POD_NAME
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
         - name: A8_ELASTICSEARCH_SERVER
           value: $(LOGSERVER_SERVICE_HOST):9200
-        - name: A8_CONTROLLER_POLL
-          value: 5s
```
Comment (rshriram) [inline testing/kubernetes/bookinfo.yaml]: Let's remove elastic search as well. I don't think we have Gremlin assertion checks in the kubectl version.

Comment (rshriram) [reply testing/kubernetes/bookinfo.yaml]: That would mean setting A8_LOG to false

File: testing/kubernetes/install-kubernetes.sh [hunk 2]
```diff
@@ -20,21 +20,21 @@ export K8S_VERSION="v1.5.2"
...
     --name=kubelet \
     gcr.io/google_containers/hyperkube-${ARCH}:${K8S_VERSION} \
     /hyperkube kubelet \
+        --containerized \
         --hostname-override=127.0.0.1 \
         --api-servers=http://localhost:8080 \
         --config=/etc/kubernetes/manifests \
-        --cluster-dns=10.0.0.10 \
-        --cluster-domain=cluster.local \
         --allow-privileged --v=2
```
Comment (rshriram) [inline testing/kubernetes/install-kubernetes.sh:46]: Why remove the dns?

**Review by rshriram — COMMENTED** [review]
I will let @elevran comment.

File: testing/kubernetes/bookinfo.yaml (hunk 1 above)
Comment (elevran) [reply testing/kubernetes/bookinfo.yaml]: Please see #514 for changed deployment files, integrating Shriram's comments as well.
I've ran through the entire walkthrough which can be used to confirm the testing scripts, if needed.

Comment (elevran) [reply testing/kubernetes/bookinfo.yaml]: The above changes are needed across all YAML files as in #514

File: testing/test-scripts/k8s/default_routes.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 IBM Corporation
```
Comment (elevran) [inline testing/test-scripts/k8s/default_routes.yaml]: we were working on these concurrently ;-(
May not be needed after a merge of #514 (which matches the bookinfo walkthrough files)

**Review by elevran — COMMENTED** [review]
has overlap with #514, consider including only testing changes for this PR, not the example code

**Review by elevran — CHANGES_REQUESTED** [review]
Travis failure needs to be fixed as well

File: testing/kubernetes/install-kubernetes.sh (hunk 2 above)
Comment (craigyam) [reply testing/kubernetes/install-kubernetes.sh:46]: Saw some problems with the go http client timing out.  Removed the lines (to match what the previous config was) and they went away.

File: .gitignore [hunk 4]
```diff
@@ -24,6 +24,7 @@ examples/docker-controlplane.yaml
 examples/docker-helloworld.yaml
 examples/docker-helloworld-default-route-rules.json
 examples/docker-helloworld-v1-v2-route-rules.json
```
Comment (elevran) [inline .gitignore]: can probably use docker-helloworld*.{json,yaml} as well

**Review by elevran — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (craigyam) [reply .gitignore]: Done

**Comment by rshriram** [comment]
@craigyam feel free to merge when you see fit.

**Review by elevran — APPROVED** [review]
LGTM.

**Pull request merged**